docs/source/configuration/overview.mdx

```diff
@@ -9,11 +9,11 @@ import RedisTLS from '../../shared/redis-tls.mdx'
 Learn how to customize the behavior of your Apollo Router with environment variables, command-line commands and options, and YAML file configuration.
 
 ## Environment variables
+Most environment variables are also documented with their [Command-line options](#command-line-options) counterpart. However, there are some options that are only available to configure as environment variables.
```

Comment:
This addition confuses me a bit since I don't see the CLI counterpart in the table below.

Also, we should specify which options are only available via env variable.

Reply:
> I don't see the CLI counterpart in the table below.

That is in the next section below `## Command-line options`

> Also, we should specify which options are only available via env variable.

That is supposed to be this section here.

Reply:
Got it, would suggest this to clarify then:

```suggestion

You can set most environment variables via their [command-line option](#command-line-options) counterpart. The following values are only configureable as environment variables:
```